<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="0" failures="1" skipped="0" tests="13" time="10.995" timestamp="2022-05-19T13:57:23.719274" hostname="Steamship.local"><testcase classname="tests.utils.test_enums" name="test_enum_serialization_behavior" time="0.000" /><testcase classname="tests.app.test_response" name="test_text_response" time="0.001" /><testcase classname="tests.app.test_response" name="test_dict_response" time="0.000" /><testcase classname="tests.app.test_response" name="test_resp_response" time="0.000" /><testcase classname="tests.base.test_binary_utils" name="test_dump_string" time="0.000" /><testcase classname="tests.base.test_binary_utils" name="test_dump_json" time="0.001" /><testcase classname="tests.base.test_binary_utils" name="test_dump_raw_data" time="0.000" /><testcase classname="tests.base.test_task" name="test_background_task_call" time="3.327" /><testcase classname="tests.base.test_task" name="test_task_update" time="2.224" /><testcase classname="tests.client.test_client" name="test_connect" time="0.001" /><testcase classname="tests.client.test_client" name="test_user" time="0.011" /><testcase classname="tests.client.operations.test_blockify" name="test_file_upload_then_parse" time="4.962" /><testcase classname="tests.client.operations.test_embed" name="test_basic_embeddings" time="0.054"><failure message="pydantic.error_wrappers.ValidationError: 1 validation error for PluginInstance&#10;hosting_platform&#10;  value is not a valid enumeration member; permitted: 'lambda', 'ecs' (type=type_error.enum; enum_values=[&lt;PlatformType.LAMBDA: 'lambda'&gt;, &lt;PlatformType.ECS: 'ecs'&gt;])">def test_basic_embeddings():
        client = get_steamship_client()
&gt;       plugin_instance = PluginInstance.create(client, plugin_handle=_TEST_EMBEDDER).data

client/operations/test_embed.py:39: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
../src/steamship/data/plugin_instance.py:121: in create
    return client.post(
../src/steamship/base/client.py:357: in post
    return self.call(
../src/steamship/base/client.py:312: in call
    data = expect.from_dict(response_data["data"], client=self)
../src/steamship/data/plugin_instance.py:83: in from_dict
    return PluginInstance(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

&gt;   ???
E   pydantic.error_wrappers.ValidationError: 1 validation error for PluginInstance
E   hosting_platform
E     value is not a valid enumeration member; permitted: 'lambda', 'ecs' (type=type_error.enum; enum_values=[&lt;PlatformType.LAMBDA: 'lambda'&gt;, &lt;PlatformType.ECS: 'ecs'&gt;])

pydantic/main.py:331: ValidationError</failure></testcase></testsuite></testsuites>